### PR TITLE
Adding new RBAC for LPSRE team (merge of CS/MT SRE teams)

### PR DESCRIPTION
**Pull request by npecka**
Signed-off-by: npecka <zangoose107@gmail.com>

### What type of PR is this?
_(feature/documentation)_

### What this PR does / why we need it?
CS & MT SRE teams have merged and need their roles combined.

### Which Jira/Github issue(s) this PR fixes?
https://issues.redhat.com/browse/MTSRE-780

### CODE REVIEW

File: deploy/backplane/lpsre/02-cssre-monitoring-rolebinding.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (jessesarn) [inline deploy/backplane/lpsre/02-cssre-monitoring-rolebinding.yml:1]: File should be renamed with lpsre instead of cssre.

Comment (npecka) [reply deploy/backplane/lpsre/02-cssre-monitoring-rolebinding.yml:1]: Oh yep thanks for the catch

File: deploy/backplane/lpsre/10-lpsre-addon-operator-olm-apis.ClusterRole.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```
Comment (dustman9000) [inline deploy/backplane/lpsre/10-lpsre-addon-operator-olm-apis.ClusterRole.yml:2]: Can this be narrowed down to a specific namespace? Do you really need cluster-wide permissions?

**Comment by dustman9000** [comment]
Verified this is a merge of the CSSRE and MTSRE existing roles. There will be a follow up PR to remove those roles once this is verified.
/lgtm
/approve

**Pull request merged**